metadataservice/server/engine.py

```diff
@@ -300,8 +305,9 @@ def post(self):
                 bulk.execute()
             except pymongo.errors.BulkWriteError as err:
                 raise tornado.web.HTTPError(500, str(err))
-            database.event.create_index([('time', pymongo.ASCENDING),
-                                         ('descriptor', pymongo.ASCENDING)])
+            database.event.create_index([('time', pymongo.DESCENDING),
+                                         ('descriptor', pymongo.DESCENDING)])
+            database.event.create_index([('time', pymongo.DESCENDING)], unique=True)
```

Comment:
No, it shouldn't be. Also, it shows up in two lines. Bottom one should be uid, that's why it is unique
PS: still getting a hang of emacs copy paste. Too focused on the key combos rather than what i am pasting